examples/kubernetes/ceph-osd-v1-ds.yaml

```diff
@@ -17,9 +17,12 @@ spec:
       nodeSelector:
         node-type: storage
       volumes:
-        - name: devices
-          hostPath:
-            path: /dev
+# Comment back in to use direct device access.
+# Note that this will also require enabling privileged access and
```

Comment:
I don't understand, isn't that direct device access, no matter whether using it as data or journal drive?

I'm happy to change the comment and would welcome some suggestion if the general idea to run non-privileged by default by itself is acceptable and has a chance of being merged.